client-side/src/components/statistics/graphsUtils.js

```diff
@@ -41,3 +41,23 @@ export function getWebsites(websites, users, user, dateStart, dateEnd) {
 
     return website;
 }
+
+export function getVisitedWebsitesByDate(users, user, date) {
+    const currentUser = users?.data?.users?.find(u => u.email === user.email);

```

Comment:
why do you need this search in users data if you send the currentuser also?
other option: think about taking the current user from the redux

Reply:
so please go to the peace of code that insert the user to the redux and check there which data is saved to the redux. if you are missing data there, add it to the redux

Reply:
see what I write you in slack

Reply:
I answered you.